tests/python_client/testcases/test_delete.py

```diff
@@ -1064,3 +1068,579 @@ def test_delete_sealed_only(self):
         collection_w.delete(expr)
 
         collection_w.query(expr, check_task=CheckTasks.check_query_empty)
+
+class TestDeleteString(TestcaseBase):
+    """
+    Test case of delete interface with string 
+    """
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_delete_entities_repeatedly_with_string(self):
```

Comment:
Do we have the case covering the scenario that delete using the same expression twice before query or search?

Reply:
Not sure what the significance is